src/subscription_manager/entcertlib.py

```diff
@@ -108,10 +108,6 @@ def perform(self) -> "EntCertUpdateReport":
         log.info("certs updated:\n%s", self.report)
         self.syslog_results()
 
-        # We call EntCertlibActionInvoker.update() solo from
-        # the 'attach' cli instead of an ActionClient. So
```

Comment:
I removed the comment, because “'attach' cli” seemed to refer to the CLI command.

Reply:
except the 'attach' reference, the rest of the comment is actually valid, and dropping it would make the below `refresh()` call without explanation

Reply:
Ok! I‘ll put the comment back except the `attach` command reference.